doc/design.md

```diff
@@ -371,6 +371,17 @@ Covers:
 
 Needs: impl
 
+#### Validating the Release Date
+`dsn~validating-release-date~1`
+
+During validation RD checks if the release date is up-to-date. If it is not equal to today's date, RD prints a warning message and fails the validation.
```

Comment:
I think we had this feature already.
There also once was the feature to fix it. But as far as I remember that dies because of branch protection rules.

Reply:
Maybe it's worth waiting for @AnastasiiaSergienko to answer why we removed that feature